### PR TITLE
Hello world example

### PR DESCRIPTION
**Pull request by jtravee**
Closes [#1310](https://github.com/rancher-sandbox/rancher-desktop/issues/1310).

### CODE REVIEW

File: docs/tutorials/hello-world-example.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+CMD ["echo", "Hello, John!"]
+```
+
+**Build and run the image for verification purposes:**
+
+<Tabs groupId="container-runtime">
+  <TabItem value="nerdctl" default>
+
+```
+nerdctl build -t John/hello:1.0 .
```
Comment (gunamata) [inline docs/tutorials/hello-world-example.md:32]: **J**ohn/hello needs to be fully lower case and all other commands need to be changed accordingly

File: sidebars.js [hunk 2]
```diff
@@ -31,7 +35,8 @@ const sidebars = {
       items: [
         "tutorials/vs-code-docker", 
         "tutorials/vs-code-remote-containers", 
-        "tutorials/rancher-ui-on-rancher-desktop"
+        "tutorials/rancher-ui-on-rancher-desktop",
+        "tutorials/hello-world-example",
```
Comment (gunamata) [inline sidebars.js:39]: Better to move hello-world-page as the first item under `Tutorial`

Comment (jtravee) [reply sidebars.js]: Updated as suggested.

File: docs/tutorials/hello-world-example.md (hunk 1 above)
Comment (jtravee) [reply docs/tutorials/hello-world-example.md]: Updated as suggested.

**Review by gunamata — APPROVED** [review]

**Pull request merged**